libraries/GCS_MAVLink/GCS_Signing.cpp

```diff
@@ -65,6 +65,13 @@ bool GCS_MAVLINK::signing_key_load(struct SigningKey &key)
  */
 void GCS_MAVLINK::handle_setup_signing(const mavlink_message_t *msg)
 {
+    // setting up signing key when armed generally not useful /
+    // possibly not a good idea
+    if (hal.util->get_soft_armed()) {
+        hal.console->printf("Won't setup signing when armed");
```

Comment:
I suggest a STATUSTEXT, as the user may be operating over telem
also, msg could be "ERROR: must disarm to setup signing"

Reply:
I was following convention in that function; should we change that convention?